<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TokenListShowDialog</class>
 <widget class="QDialog" name="TokenListShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">List ZRC721 Tokens</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutMain">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <layout class="QGridLayout" name="gridLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayoutLoadCurrent">
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutLoad">
       <item>
        <widget class="QLabel" name="labelLoad">
         <property name="text">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLoadCurrent">
         <property name="text">
           <string></string>
          </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayoutFooter">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TokenListShowDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
